i18n/en.json

```diff
@@ -492,18 +498,30 @@
   "admin.experimental.experimentalEnableHardenedMode.title": "Enable Hardened Mode",
   "admin.experimental.experimentalHideTownSquareinLHS.desc": "When true, hides Town Square in the left-hand sidebar if there are no unread messages in the channel. When false, Town Square is always visible in the left-hand sidebar even if all messages have been read.",
   "admin.experimental.experimentalHideTownSquareinLHS.title": "Town Square is Hidden in Left-Hand Sidebar",
+  "admin.experimental.experimentalLdapGroupSync.desc": "When true, enables **AD/LDAP Group Sync** configurable under **Access Controls > Groups**. See [documentation](!https://docs.mattermost.com/deployment/ldap-group-sync.html) to learn more.",
```

Comment:
Propose changing doc link to a permalink https://mattermost.com/pl/default-ldap-group-sync

(Have created a ticket for marketing to add)